bear/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer: Moritz Lipp <mlq@pwmt.org>
+
+pkgname=bear
+pkgver=2.1.5
+pkgrel=1
+pkgdesc="tool to generate compilation database for clang tooling"
+arch=('i686' 'x86_64')
```

Comment:
These are the only two possible values in the `arch` array of a PKGBUILD. ARM is not supported on Arch Linux.